velox/exec/benchmarks/HashJoinListResultBenchmark.cpp

```diff
@@ -260,7 +260,10 @@ class HashTableListJoinResultBenchmark : public VectorTestBase {
     if (addExtraValue) {
       data[0] = params_.extraValue;
     }
-    std::random_shuffle(data.begin(), data.end());
+
+    std::random_device rd;
+    std::mt19937 g(rd());
```

Comment:
Put the RNG as a field of the class and initialize it in constructor, because `rd()` is expensive and this method is called repeatedly.